### ISSUE TITLE
Fix npm run test and friends

### ISSUE THREAD

**Issue by joshnesbitt** [sky-uk/blackjack-toolbelt#34]
When you try and run the toolbelt from NPM, require paths get messed up. This is because NPM is trying to use the toolbelt installed via NPM instead of your global one. We need to force it to use the global one for now I think.

**Comment by joshnesbitt** [comment]
One way around this would be to symlink the toolbelt to `/usr/local/bin/blackjack` and reference that absolute script from `package.json`, but this feels bad.

**Comment by yjcxy12** [comment]
I believe this only happens if you install toolbelt as dependency in child project. Is it the correct way to use this? To install it both globally and locally, and use global commands?

**Comment by joshnesbitt** [comment]
Yup so the idea is you install the toolbelt globally and use that binary, and install it locally in the project to pull in dependencies. The issue is when NPM wants you to use the locally installed binary, as it messes with some paths. I think we can fix this with some magic in the `constants.js` file to check whether it's running from within the context of NPM, but I've not had chance to get a fix working yet. I [did have a go](https://github.com/sky-uk/blackjack-toolbelt/commit/7aad450dec0f634433cda6b507bd2d49d17cbc71), but it was a bit buggy.
